include/z64map.h

```diff
@@ -0,0 +1 @@
+#define FLOOR_INDEX_MAX 4
```

Comment:
```suggestion
#ifndef Z64MAP_H
#define Z64MAP_H

#define FLOOR_INDEX_MAX 4

#endif
```
Don't forget the header-guards

Reply:
Whoops, haven't made a C header in over an age. Thanks for the reminder.